packages/components/src/RichTooltip/RichTooltip.component.js

```diff
@@ -0,0 +1,123 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames';
+import { Actions } from '../Actions';
+import CircularProgress from '../CircularProgress';
+import Icon from '../Icon';
+import Inject from '../Inject';
+import theme from './RichTooltip.scss';
+
+export function RichTooltipContent(props) {
+	let content;
+
+	if (props.loading) {
+		content = (
+			<div className={theme['tc-center-content']}>
+				<CircularProgress />
+			</div>
+		);
+	}
+
+	if (props.error) {
+		content = (
+			<div className={classNames(theme['tc-center-content'], theme['tc-error-wrapper'])}>
+				<Icon name="talend-warning" className={theme['tc-center-icon']} />
+				<div className={theme['tc-error-body']}>
+					<h1>Whoops !</h1>
+					<div>{props.error}</div>
+				</div>
+			</div>
+		);
+	}
+
+	if (typeof props.Content === 'string' && !(props.error || props.loading)) {
+		content = props.Content;
+	}
+
+	if (content) {
+		return <RichTooltip.Body className={props.className} Content={content} />;
+	}
+
+	return Inject.getReactElement(props.getComponent, props.Content);
+}
+
+RichTooltipContent.propTypes = {
+	className: PropTypes.string,
+	getComponent: PropTypes.func,
+	loading: PropTypes.bool,
+	Content: Inject.getReactElement.propTypes,
+	error: PropTypes.string,
+};
+
+export function RichTooltipBody(props) {
+	// fix ie11 flexbox align-items: center with min-height https://stackoverflow.com/a/47180276
+	return (
+		<div
+			className={classNames(props.className, {
+				[theme['tc-tooltip-body']]: true,
+				'tc-tooltip-body': true,
+			})}
+		>
+			<div className={classNames(theme['tc-tooltip-fix-ie'], 'tc-tooltip-fix-ie')}>
+				{props.Content}
+			</div>
+		</div>
+	);
+}
+
+RichTooltipBody.propTypes = {
+	className: PropTypes.string,
+	Content: RichTooltipContent.propTypes.Content,
+};
+
+export function RichTooltipHeader(props) {
+	return (
+		<header className={classNames(theme['tc-tooltip-header'], 'tc-tooltip-header')}>
+			<h1 title={props.title}>{props.title}</h1>
+			<Actions actions={props.right} />
+		</header>
+	);
+}
+
+RichTooltipHeader.propTypes = {
+	title: PropTypes.string.isRequired,
+	right: PropTypes.array,
+};
+
+export function RichTooltipFooter(props) {
+	return (
+		<footer className={classNames(theme['tc-tooltip-footer'], 'tc-tooltip-footer')}>
+			<Actions actions={props.left} />
+			<Actions
+				className={classNames(theme['tc-tooltip-right-actions'], 'tc-tooltip-right-actions')}
+				actions={props.right}
+			/>
+		</footer>
```

Comment:
Why still using actions ? The guidlines shows that there are 3 kind footers, can we make a dedicated components and inject a custom one if we need.
The props will be simpler.